src/bench/logging.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2020-2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
```

Comment:
don't add a file that bitcoin modified; src/bench/logging.cpp

Find what PR this PR depends on that originally added this file.